go/vt/sqlparser/parse_test.go

```diff
@@ -4932,6 +4944,12 @@ var (
 	}, {
 		input:  "insert into a select * into @a from b",
 		output: "INTO clause is not allowed at position 38 near 'b'",
+	}, {
+		input:  "create table t (id int primary key, col1 int SRID 0)",
+		output: "cannot define SRID for non spatial types at position 53 near '0'",
+	}, {
+		input:  "create table t (id int primary key, col1 int SRID -1)",
```

Comment:
Looks like this test is specifically for invalid, negative SRIDs? If so, you should change the column type to POINT or GEOMETRY or something so that it parses with a valid type and doesn't have any chance of returning the same error as on the test above. Plus, that would also make the test just a little bit more explicit about what it's testing.